fmriprep/workflows/bold/registration.py

```diff
@@ -685,12 +679,27 @@ def init_fsl_bbr_wf(use_bbr, bold2t1w_dof, name='fsl_bbr_wf'):
 
     workflow.connect([
         (inputnode, wm_mask, [('t1w_dseg', 'in_seg')]),
-        (inputnode, flt_bbr, [('in_file', 'in_file'),
-                              ('t1w_brain', 'reference')]),
+        (inputnode, flt_bbr, [('in_file', 'in_file')]),
         (flt_bbr_init, flt_bbr, [('out_matrix_file', 'in_matrix_file')]),
-        (wm_mask, flt_bbr, [('out', 'wm_seg')]),
     ])
 
+    if sloppy is True:
+        downsample = pe.Node(niu.Function(
+            function=_conditional_downsampling, output_names=["out_file", "out_mask"]),
+            name='downsample')
+        workflow.connect([
+            (inputnode, downsample, [("t1w_brain", "in_file")]),
+            (wm_mask, downsample, [("out", "in_mask")]),
+            (downsample, flt_bbr, [('out_file', 'reference'),
+                                   ('out_mask', 'wm_seg')]),
```

Comment:
With the utilization of cached, high-resolution anatomical derivatives, the time it took for flirt/bbr to run exploded. With this patch, I minimized flirt/bbr from ~5min to ~11sec (on my desktop, it must be a bit longer on CircleCI). Some other nodes have extended their duration a bit as when used pre-computed anatomical derivatives, but I think now the timing is reasonable.

I used this simple script: https://gist.github.com/oesteban/0c17459936deb18ac81377db4b4763c2 to measure times. We should consider running something like that in fmriprep to get some stats about execution etc.